Lib/test/test_sys_settrace.py

```diff
@@ -2064,6 +2064,40 @@ def test_jump_simple_backwards(output):
         output.append(1)
         output.append(2)
 
+    @jump_test(1, 4, [5])
+    def test_jump_is_none_forwards(output):
+        x = None
+        if x is None:
+            output.append(3)
+        else:
+            output.append(5)
+
+    @jump_test(6, 5, [3, 5, 6])
+    def test_jump_is_none_backwards(output):
+        x = None
+        if x is None:
+            output.append(3)
+        else:
+            output.append(5)
+        output.append(6)
+
+    @jump_test(1, 4, [5])
+    def test_jump_is_not_none_forwards(output):
+        x = None
+        if x is not None:
+            output.append(3)
+        else:
+            output.append(5)
+
+    @jump_test(6, 2, [5, 5, 6])
```

Comment:
Nitpick: this still doesn't fail on main for me. This does, though:

```suggestion
    @jump_test(6, 5, [5, 5, 6])
```

I think you have to jump into the `else` to trigger this bug. It's pretty subtle!

Reply:
Oh yeah, I must have missed that one! Thanks!